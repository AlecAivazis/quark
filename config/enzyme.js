// external imports
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import browserEnv from 'browser-env'

// add a request animation frame polyfill
import raf from 'raf'
raf.polyfill()

// enable the react 16 adapter for enzyme
Enzyme.configure({ adapter: new Adapter() })

// add various browser mocks
browserEnv()
