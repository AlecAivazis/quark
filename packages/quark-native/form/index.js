import _TextInput from './TextInput';
export { _TextInput as TextInput };