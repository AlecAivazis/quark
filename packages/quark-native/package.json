{
    "name": "quark-native",
    "version": "0.1.4",
    "module": "index.js",
    "main": "index.js",
    "dependencies": {
        "quark-core": "^0.1.4"
    }
}
