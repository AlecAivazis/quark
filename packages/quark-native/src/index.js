// @flow
export * from './components'
export * from 'quark-core'
