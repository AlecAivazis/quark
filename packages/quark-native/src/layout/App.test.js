// external imports
import React from 'react'
import { Text } from 'react-native'
import { mount } from 'enzyme'
// local imports
import { App } from 'quark-native'

it('renders', () => expect(mount(<App />)).toMatchSnapshot())
