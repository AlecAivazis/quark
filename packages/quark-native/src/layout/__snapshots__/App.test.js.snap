// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<App>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <div
      className="rn-alignItems-1oszu61 rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-6koalj rn-flex-13awgt0 rn-flexGrow-1m1wadx rn-flexShrink-1awmn5t rn-flexDirection-eqz5dr rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-minHeight-ifefl9 rn-minWidth-bcqeeo rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-position-bnwqim rn-zIndex-1lgpqti"
    />
  </View>
</App>
`;
